src/fluxo.object_store.js

```diff
@@ -117,6 +151,10 @@ export default class {
   unsetAttribute  (attribute, options) {
     options = options || {};
 
+    if (this.data[attribute] instanceof this.constructor) {
+      this.stopListenStoreAttribute(attribute);
+    }
+
```

Comment:
`this.constructor` will be always `Fluxo.ObjectStore`?

If I have a store A inheriting from `Fluxo.ObjectStore` and then a store B inheriting from A, the constructor of B will be `Fluxo.ObjectStore` or `A`?